library/include/flipper/posix/posix.h

```diff
@@ -30,8 +31,8 @@
 #define __use_wdt__
 
 /* Declare the LF_VAR and LF_FUNC types for this platform. */
-#define LF_VAR __attribute__((section("__DATA,.lf.vars")))
-#define LF_FUNC __attribute__((section("__TEXT,.lf.funcs")))
```

Comment:
This will break macOS. Need this here for clang because of how section names work.

Reply:
a6f261d3a62e32bda103afc13fc9d5ed8b410237 should fix this.